scipy/stats/tests/test_stats.py

```diff
@@ -5023,6 +5020,64 @@ def test_trim_bounds_error(self, trim):
             stats.ttest_ind([1, 2], [2, 1], trim=trim)
 
 
+class Test_ttest_CI:
+    # indices in order [alternative={two-sided, less, greater},
+    #                   equal_var={False, True}, trim={0, 0.2}]
+    # reference values in order `statistic, df, pvalue, low, high`
+    # equal_var=False reference values computed with R PairedData yuen.t.test:
+    #
+    # library(PairedData)
+    # options(digits=16)
+    # a < - c(0.88236329, 0.97318744, 0.4549262, 0.97893335, 0.0606677,
+    #         0.44013366, 0.55806018, 0.40151434, 0.14453315, 0.25860601,
+    #         0.20202162)
+    # b < - c(0.93455277, 0.42680603, 0.49751939, 0.14152846, 0.711435,
+    #         0.77669667, 0.20507578, 0.78702772, 0.94691855, 0.32464958,
+    #         0.3873582, 0.35187468, 0.21731811)
+    # yuen.t.test(a, b, tr=0, conf.level = 0.9, alternative = 'l')
+    #
+    # equal_var=True reference values computed with R multicon yuenContrast:
+
+    r = np.empty(shape=(3, 2, 2, 5))
+    r[0, 0, 0] = [-0.2314607, 19.894435, 0.8193209, -0.247220294, 0.188729943]
```

Comment:
I didn't set out thinking that I would populate the array this way. `r` started out as a dictionary, and I went through a lot of iterations to figure out how to store the data compactly within the PEP8 line limit. If the reviewer would prefer for this to be refactored, please make a specific suggestion, but I would prefer to do so under the following constraints:
- One line per case
- Include sufficient precision to use default `assert_allclose` tolerances.

One thought it that I could eliminate the statistic and p-value, since in theory those are already validated elsewhere. But I done that, I would not have caught the `equal_var` discrepancy, so I think it's good to leave them in.

Reply:
The formatting looks good to me, especially given the comment above about what each slice of the array is nad how it's computed.